### ISSUE TITLE
Support listing assets without a group

### ISSUE THREAD

**Issue by benarmston** [alces-flight/flight-asset-cli#8]
We currently, support listing all assets and listing all assets with a given group.  It is not currently possible to list all assets that do not have a group. We should support that.

**Comment by WilliamMcCumstie** [comment]
WIP

**Comment by WilliamMcCumstie** [comment]
Version `0.3.0` Has made progress on this issue but the UI still needs work. Ideally the flag would be `--[no-]group [GROUP]`

```  SYNOPSIS:

    flight-asset list-assets 

  DESCRIPTION:

    Return all the assets

  OPTIONS:

    --group [GROUP] 
        Filter the assets by GROUP (or no group if omitted)
  
    --[no-]decommissioned 
        Explicitly filter the assets by the decomissioned flag:
          * --decommissioned    exclusively decommissioned
          * --no-decommissioned exclusively (re)commissio
```

**Comment by mjtko** [comment]
I'm not keen on the optional `GROUP` and `--[no-]...` approaches -- instead, I would expect something like this:

```
--group GROUP
  List only the assets in GROUP. Specify the empty string ("") to list only assets that are not in a group.

--include-decommissioned 
   Include assets that have been decommissioned.

--only-decommissioned
  List only assets that have been decommissioned.
```

**Comment by WilliamMcCumstie** [comment]
Fixed in the `0.3.1` release
